src/zenml/models/v2/base/scoped.py

```diff
@@ -524,79 +523,3 @@ def get_custom_filters(
             )
 
         return custom_filters
-
-    def apply_sorting(
```

Comment:
Why was the tag sorting removed now?

Reply:
Unfortunately, the tests showed that the concatenation of the tags alphabetically did not work as intended and tags were sorted randomly. Until we could figure out a solution, I have decided to disable sorting by tags.